server/service/src/sync/test/integration/errors.rs

```diff
@@ -95,13 +95,13 @@ mod tests {
 
         assert_matches!(
             error,
-            SyncError::RemotePushError(RemotePushError::SyncApiError(SyncApiError {
+            SyncError::SyncApiError(SyncApiError {
```

Comment:
Synchroniser now does a `get_site_info` call to get V6 central server info, so now get the unauthorised error at that step, rather than on remote push!